crates/biome_html_formatter/src/html/auxiliary/element.rs

```diff
@@ -130,10 +132,16 @@ impl FormatNodeRule<HtmlElement> for FormatHtmlElement {
                 } => {
                     write!(
                         f,
-                        [best_fitting![
-                            format_args![flat_children],
-                            format_args![expanded_children]
-                        ]]
+                        [
+                            // If the attribute group breaks, prettier always breaks the children as well.
+                            &if_group_breaks(&expanded_children).with_group_id(Some(attr_group_id)),
+                            // Otherwise, we can do whatever fits best for the children.
+                            &if_group_fits_on_line(&best_fitting![
+                                format_args![flat_children],
+                                format_args![expanded_children]
+                            ])
+                            .with_group_id(Some(attr_group_id))
+                        ]
```

Comment:
I can't seem to figure out why this sometimes doesn't emit content. The ordering of these lines also appears to change the outcome.